tests/countries/test_bonaire_sint_eustatius_and_saba.py

```
@@ -0,0 +1,268 @@
+            "2025-04-26",
+            "2031-04-26",
+            "2036-04-26",
+        )
+        self.assertNoHoliday(
+            "2014-04-27",
+            "2025-04-27",
+            "2031-04-27",
+            "2036-04-27",
+        )
+
+    def test_rincon_day(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)